src/backend/templates/calendar.twig

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Hackerspace Calendar</title>
+
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/paper-css/0.3.0/paper.css">
+	<link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+	<style>	
+	page { size: A2 }
+
+  body.A2           .sheet { width: 420mm; height: 594mm }

```

Comment:
I typed code like that, because I extended [paper.css](https://cdnjs.cloudflare.com/ajax/libs/paper-css/0.3.0/paper.css) and tried to follow author's style. Maybe it will be better to download that asset and move this code over there.